src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -236,7 +236,10 @@ export default App.extend(extend({
     });
     attachment.upload(file);
 
-    Radio.request('ws', 'add', attachment);
+    this.listenTo(attachment, 'upload:success', uploadedAttachment => {
```

Comment:
we should combine this with 244 `this.listenTo(attachment, {}`

Reply:
Yep good call, updated to that 👍